backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -1,21 +1,27 @@
 import datetime
-from typing import Mapping
+from typing import Annotated, Mapping
 
-from msgspec import UNSET, Struct, UnsetType, field
+from ape_pie import APIClient
+from msgspec import UNSET, Meta, Struct, UnsetType
+from rest_framework.request import Request
 from openbeheer.api.views import ListView
 from openbeheer.types._zgw import ZGWResponse
 from openbeheer.types.ztc import Status, ValidatieFout, VertrouwelijkheidaanduidingEnum
 from openbeheer.types import OBPagedQueryParams, OBField, OBOption
 from drf_spectacular.utils import extend_schema, extend_schema_view
 
 
-class ZaaktypenGetParametersQuery(OBPagedQueryParams, kw_only=True):
-    catalogus: int | None = None
-    datum_geldigheid: str | None = field(name="datumGeldigheid", default=None)
-    identificatie: str | None = None
+class ZaaktypenGetParametersQuery(OBPagedQueryParams, kw_only=True, rename="camel"):
+    catalogus: Annotated[
+        str | UnsetType, Meta(description="UUID part of the catalogus URL")
+    ] = UNSET  # frontend uuid.UUID, backend url
```

Comment:
I don't know how I feel about this, since all the other query params are the same as in OZ except the catalogus :thinking: 
Maybe this struct could have a method `convert_to_catalogus_url` that could do the logic that is now in `parse_query_params`?